docs/index.rst

```diff
@@ -41,14 +42,19 @@ Available Solidity Integrations
 	Qt based IDE for designing, debugging and testing solidity smart contracts.
 
 * `Package for SublimeText — Solidity language syntax <https://packagecontrol.io/packages/Ethereum/>`_
-	Solidity syntax highlight for SublimeText editor.
+	Solidity syntax highlighting for SublimeText editor.
 
 * `Atom Solidity package <https://github.com/gmtcreators/atom-solidity/>`_
 	Plugin for the atom editor that features syntax highlighting, compilation and a runtime environment (requires backend node).
 
 * `Visual Studio Code extension <http://juan.blanco.ws/solidity-contracts-in-visual-studio-code/>`_
 	Solidity plugin for Microsoft Visual Studio Code that includes syntax highlighting and the Solidity compiler.
-	
+
+* `Emacs Solidity <https://github.com/ethereum/emacs-solidity/>`_
+	Plugin for the emacs editor providing syntax highlighting and compilation error reporting.
+
+* `VIM Solidity <https://github.com/ethereum/vim-solidity/>`_
```

Comment:
I already PR'd with the updated keyword list according to [Token.h](https://github.com/ethereum/solidity/blob/develop/libsolidity/parsing/Token.h). The current version has everything except for `fixed`/`ufixed`, which I'm waiting on to be pulled in, at the moment.